### ISSUE TITLE
Is it possible to omit logs of K-means clustering?

### ISSUE THREAD

**Issue by Solacex** [facebookresearch/faiss#1481]
# Summary

# Platform

OS: Linux

Faiss version: 1.6.1

I want to know if we could omit these logs (Output below) about k-means during training. 

# Reproduction instructions

Codes:
        feat = feat.cpu().numpy()
        id2lbl = {}
        d = feat.shape[1]
        k = num_centers
        kmeans = faiss.Kmeans(d, k, niter=50, verbose=True,spherical=True, min_points_per_centroid=10)
        kmeans.train(feat)
        center = kmeans.centroids
        D, I = kmeans.index.search(feat, 1)
        center = torch.from_numpy(center).cuda()
        I= torch.from_numpy(I).cuda()
        center = F.normalize(center, p=2 , dim=-1)
        return center, I

Output:
Sampling a subset of 2560 / 50000 for training
Clustering 2560 points in 512D to 10 clusters, redo 1 times, 50 iterations
  Preprocessing in 0.13 s
  Iteration 49 (0.06 s, search 0.01 s): objective=2548.59 imbalance=1.014 nsplit=0
Sampling a subset of 2560 / 50000 for training
Clustering 2560 points in 512D to 10 clusters, redo 1 times, 50 iterations
  Preprocessing in 0.13 s
  Iteration 49 (0.07 s, search 0.01 s): objective=2550.31 imbalance=1.058 nsplit=0
Sampling a subset of 3840 / 50000 for training
Clustering 3840 points in 512D to 15 clusters, redo 1 times, 50 iterations
  Preprocessing in 0.13 s
  Iteration 49 (0.10 s, search 0.02 s): objective=3823.62 imbalance=1.037 nsplit=0
Sampling a subset of 3840 / 50000 for training
Clustering 3840 points in 512D to 15 clusters, redo 1 times, 50 iterations
  Preprocessing in 0.13 s
  Iteration 49 (0.09 s, search 0.02 s): objective=3826.03 imbalance=1.080 nsplit=0
Sampling a subset of 5120 / 50000 for training
Clustering 5120 points in 512D to 20 clusters, redo 1 times, 50 iterations
  Preprocessing in 0.13 s
  Iteration 49 (0.14 s, search 0.04 s): objective=5098.76 imbalance=1.043 nsplit=0
Sampling a subset of 5120 / 50000 for training
Clustering 5120 points in 512D to 20 clusters, redo 1 times, 50 iterations
  Preprocessing in 0.13 s
  Iteration 49 (0.14 s, search 0.04 s): objective=5101.97 imbalance=1.047 nsplit=0
Sampling a subset of 6400 / 50000 for training
Clustering 6400 points in 512D to 25 clusters, redo 1 times, 50 iterations
  Preprocessing in 0.13 s
  Iteration 49 (0.17 s, search 0.06 s): objective=6373.98 imbalance=1.059 nsplit=0
Sampling a subset of 6400 / 50000 for training
Clustering 6400 points in 512D to 25 clusters, redo 1 times, 50 iterations
  Preprocessing in 0.13 s
  Iteration 49 (0.18 s, search 0.05 s): objective=6378.08 imbalance=1.063 nsplit=0
Sampling a subset of 7680 / 50000 for training
Clustering 7680 points in 512D to 30 clusters, redo 1 times, 50 iterations
  Preprocessing in 0.13 s

**Comment by mdouze** [comment]
??? you call it with verbose=True

**Comment by Solacex** [comment]
..... Sorry for that, my fault.
